### DISCUSSION TITLE
gradients dont work on firefox

### DISCUSSION THREAD

**Discussion by zxq5-dev** [tailwindlabs/tailwindcss#10996] [Help]
**What version of Tailwind CSS are you using?**

the latest version as of 12/04/23 installed using npm

**What build tool (or framework if it abstracts the build tool) are you using?**

trunk (rust webserver tool)

**What version of Node.js are you using?**

latest from arch repos

**What browser are you using?**

firefox and opera

**What operating system are you using?**

linux

**Reproduction URL**

a reproduction is anything involving gradients on firefox whether it be text or the background, it just wont render.

**Describe your issue**

gradient effects such as (bg-gradient) dont work on firefox, i tested the same code on opera and it worked completely fine, im not sure if this is a tailwind issue or a firefox issue. i had issues with two things. my elements just dissappear if i set my background to a gradient but only on firefox

EDIT: updated to the latest version of firefox, no changes

**Comment by adamwathan** [comment]
Hey! This works as expected as far as I can tell:

https://play.tailwindcss.com/fBPfrTXw1J

<img width="1379" alt="image" src="https://user-images.githubusercontent.com/4323180/231522282-6988aa4b-7ea6-4a76-9bed-f2568560be81.png">

Can you provide a reproduction and specific version of Firefox?

**Reply by KauanCurbani** [reply]
using in my text dosnt work

```
 <span className="bg-gradient-to-r to-[#FF1AA3] from-[#FFCD1A] text-transparent bg-clip-text relative block">
            look at me.
 </span>
```

**Reply by wongjn** [reply]
You'd want to use `class` attribute – `className` is React nomenclature. See [this Tailwind Play of it working](https://play.tailwindcss.com/sol23dotjf) with `class`.
